io.github.chris2511.xca.yaml

```diff
@@ -0,0 +1,68 @@
+app-id: io.github.chris2511.xca
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
```

Comment:
```suggestion
  - --socket=fallback-x11
```

Reply:
I changed it to use fallback-x11 as requested.